### DISCUSSION TITLE
request log与session log关联

### DISCUSSION THREAD

**Discussion by beebol** [bfenetworks/bfe#1200] [General]
BFE access日志中 request log能显示出客户端的端口吗？session log是有的，如何对应关系？
session中是有sessionId的，但在request log和session log都没有打印，所以没法关联。

**Comment by YuncyYe** [comment]
1.目前在request log中没有输出客户端的端口
2.当前request log, session log均没有输出session id，没有可靠的方式建立关联

如果需要在request log中输出客户端的端口，建议修改代码然后发pr。
为了避免影响已有工具，可以新增字段输出客户端的端口。
